homeassistant/components/homematic/cover.py

```diff
@@ -92,7 +92,9 @@ def current_cover_tilt_position(self):
         if "LEVEL_2" not in self._data:
             return None
 
-        return int(self._data.get("LEVEL_2", 0) * 100)
+        if (position := self._data.get("LEVEL_2", 0)) is None or position == "":
+            return None
+        return int(position * 100)
```

Comment:
This can be combined with the previous `if` statement.

```py
        if "LEVEL_2" not in self._data:
            return None

        if (position := self._data.get("LEVEL_2", 0)) is None or position == "":
            return None
        return int(position * 100)
```

Becomes:

```py
        if not (position := self._data.get("LEVEL_2", 0)):
            return None
        return int(position * 100)
```